test/Common/TestHelpers.cs

```diff
@@ -66,7 +66,8 @@ internal static class TestHelpers
             bool rollbackEntityOperationsOnExceptions = true,
             int entityMessageReorderWindowInMinutes = 30,
             string exactTaskHubName = null,
-            bool addDurableClientFactory = false)
+            bool addDurableClientFactory = false,
+            Type[] types = null)
```

Comment:
We may want to hold on merging this code unless we figure out how to do the in-flight tests that actually need this functionality.